### ISSUE TITLE
Can't use import subcommand

### ISSUE THREAD

**Issue by tshchensek** [beelit94/python-terraform#8]
Hi,

With Python 2.7.10, the compiler can't parse the `import` subcommand:
```
Python 2.7.10 (default, Feb  6 2017, 23:53:20) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from python_terraform import Terraform
>>> t = Terraform()
>>> t.import('aws_instance', 'i-abc123')
  File "<stdin>", line 1
    t.import('aws_instance', 'i-abc123')
           ^
SyntaxError: invalid syntax
>>> 
```

How would you advise getting around that?

Thanks

**Comment by beelit94** [comment]
@tshchensek 
Thank you for reporting this, didn't consider this case before. I should definitely fix it.
For a quick workaround, you can simply use cmd method
`t.cmd('import', 'aws_instance', 'i-abc123')`

**Comment by beelit94** [comment]
@tshchensek 
Fix with https://github.com/beelit94/python-terraform/pull/9
Feel free to review it.

**Comment by tshchensek** [comment]
#9 LGTM, thanks for addressing this

**Comment by beelit94** [comment]
fixed in 0.8.5
